### PR TITLE
Include original traceback when re-throwing an exception inside the sensor wrapper

### PR DESCRIPTION
**Pull request by Kami**
When we re-throw a more user-friendly exception we didn't include the original stack trace which would make some issues harder to debug (it didn't include additional stack trace which would point user to the offending line in the sensor file and similar).

Caught by @enykeev.

### CODE REVIEW

**Comment by lakshmi-kannan** [comment]
lgtm.

File: st2reactor/st2reactor/container/sensor_wrapper.py [hunk 1]
```diff
@@ -264,8 +265,10 @@ def _get_sensor_instance(self):
                                                         file_path=self._file_path,
                                                         class_name=self._class_name)
         except Exception as e:
-            msg = ('Failed to load sensor class from file "%s"'
-                   ' (sensor file most likely doesn\'t exist): %s' % (self._file_path, str(e)))
+            tb_msg = traceback.format_exc()
+            msg = ('Failed to load sensor class from file "%s" (sensor file most likely doesn\'t '
+                   'exist or contains invalid syntax): %s' % (self._file_path, str(e)))
+            msg += '\n\n' + tb_msg
             raise ValueError(msg)
```
Comment (lakshmi-kannan) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: While we are on this, do you want to raise a specific exception than ValueError?

Comment (Kami) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: We could just re-throw original exception type or simply Exception.

Comment (Kami) [inline st2reactor/st2reactor/container/sensor_wrapper.py]: Also added tests - 486c4af185c052481e617d522dcff702e0d191f8, d463290b48aded5f1c973db1ce325a9d334de393.

File: st2reactor/tests/unit/test_sensor_wrapper.py [hunk 2]
```diff
@@ -107,9 +107,31 @@ def test_sensor_init_fails_file_doesnt_exist(self):
...
+        self.assertRaisesRegexp(NameError, expected_msg, SensorWrapper,
+                                pack='core', file_path=file_path,
+                                class_name='TestSensor',
+                                trigger_types=trigger_types,
+                                parent_args=parent_args)
+
+        # Verify error message also contains traceback
+        try:
+            SensorWrapper(pack='core', file_path=file_path, class_name='TestSensor',
+                          trigger_types=trigger_types, parent_args=parent_args)
```
Comment (lakshmi-kannan) [inline st2reactor/tests/unit/test_sensor_wrapper.py]: You want a self.fail('Should have thrown an exception'). Otherwise, tests might still pass on an edge case :P

Comment (Kami) [inline st2reactor/tests/unit/test_sensor_wrapper.py]: Yep, forgot to add else :)

Comment (Kami) [inline st2reactor/tests/unit/test_sensor_wrapper.py]: Actually, there is else 3 lines below :P

Comment (lakshmi-kannan) [inline st2reactor/tests/unit/test_sensor_wrapper.py]: Missed it baby one more time :D
